tinyagentos/installers/hf_multi_installer.py

```
@@ -271,17 +308,32 @@ def _cb(local_done: int, local_total: int) -> None:
-                # the byte count anyway so the aggregate progress is right.
+                # Skip files we've already downloaded — file_set_hash
+                # verification covers the final set, so a present file is
+                # a present file. Surface the byte count anyway so the
+                # aggregate progress is right.
                 downloaded_bytes += file_size
                 per_file_prev[rfilename] = file_size
                 if on_progress is not None:
                     try:
                         on_progress(downloaded_bytes, total_bytes)
                     except Exception:  # noqa: BLE001
                         pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)